src/compiler/checker.ts

```diff
@@ -34910,6 +34911,10 @@ namespace ts {
                 }
             }
 
+            if (compilerOptions.noImplicitOverride && !nodeInAmbientContext) {

```

Comment:
Does this mean we don’t catch unnecessary/incorrect `override` modifiers if `noImplicitOverride` is `false`/undefined?

```ts
// @noImplicitOverride: false
class C {
  override m() {}
}
```

I saw an earlier playground build that issued an error on `override` modifiers used without the flag turned on, but I don’t see code that does that now—was it removed? I could just be overlooking it.

Reply:
@andrewbranch I added a test for this and fixed the logic. Currently waiting for the perf tests.